wp-content/themes/ipbs-largo/less/map.less

```diff
@@ -1,55 +1,24 @@
 @import (reference) "variables.less";
 @font-map-labels: @font-description;
 
-#map-frame{
-  width: 100%;
-  height: auto;
-
-  line {
-    stroke-dashoffset: 1000;
-  }
-  .coverage {
-    pointer-events: none;
-  }
-  .coverage > g > circle {
-    opacity: 0;
-  }
-  .coverage > g > line {
-    opacity: 0;
-  }
-  .coverage > g > text {
-    opacity: 0;
-    font-family: @font-map-labels;
-    font-weight: 700;
-    font-size: 22px;
-  }
-  .heading {
-    opacity: 0;
-  }
-  .label {
-    cursor: pointer;
-  }
-  .heading text,
-  .label text {
-    fill: #515350;
-    text-transform: uppercase;
-    font-family: @font-map-labels;
-    font-weight: 700;
-  }
-  .label text {
-    font-size: 24px;
-  }
-  .heading text {
-    font-size: 20px;
-  }
-
+.ipbs-member-map-widget {
+  margin-bottom: 0!important;
   #legend {
     font-size: 16px;
     line-height: 16px;
     font-family: @font-map-labels;
     color: #221F1F;
     border: 2px solid #d1d2d3;
     padding: 0.5em;
+    width: fit-content;
```

Comment:
`fit-content` isn't widely supported at this time: https://caniuse.com/#search=fit-content

`max-content` works here, I think?

Reply:
Updated in https://github.com/INN/umbrella-ipbs/pull/83/commits/538da70a6325e3abed525fc039e22d100dfa9df2